experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -44,6 +51,25 @@ import {
 	schematizeView,
 } from "./sharedTreeView";
 
+/**
+ * Copy of data from an {@link ISharedTree} at some point in time.
+ * @alpha
+ */
+export interface TreeSnapshot {
```

Comment:
Seeing as `ISnapshotTree` and the colloquial "tree snapshot" already refer to two different things, i'd prefer to not overload this further. I don't have a better suggestion over "snapshot", but I think "ViewSnapshot" is slightly more distinctive and still fairly accurate.

Reply:
This type is the same as TreeContent, except it has stored schema instead of ViewSchema, aka the version without applying the ViewSchema. So in a way is the non-view version, so I don't think ViewSnapshot is a great name.

Since almost no one needs to refer to this, I think a long specific name might be best. Maybe SharedTreeContentSnapshot?

Reply:
That sounds good to me. Probably explicitly call out in the top-level comment that 'snapshot' is distinct from the Fluid concept of a 'snapshot', since information about how to collaborate is not preserved (the notes you have on `contentSnapshot()` are great)

Reply:
done.